scripts/workers/code_upload_submission_worker.py

```diff
@@ -67,6 +67,56 @@ def get_submission_meta_update_curl(submission_pk):
     return curl_request
 
 
+def get_static_code_upload_submission_file_curl(
+    challenge_pk, phase_pk, submission_pk, submission_path
+):
+    url = "{}/api/jobs/challenge/{}/challenge_phase/{}/submission/{}".format(
```

Comment:
@savish28 let's move this to a script `make_submission.sh` and in pre-stop hook just do `sh /path/to/make_submission.sh`